R-package/R/utils.R

```diff
@@ -19,7 +19,7 @@ lgb.is.null.handle <- function(x) {
   )
 }
 
-lgb.params2str <- function(params) {
+.lgb_params2str <- function(params) {
```

Comment:
Did this function preserve the `lgb` prefix because it's a single word?

Reply:
hmm I didn't really think about the fact that this was the only one left with `lgb_`.

Just pushed a commit removing that: https://github.com/microsoft/LightGBM/pull/6179/commits/eb0ae49449959f18895a5ccf69fb34900ca1c50c

Thanks for catching the inconsistency.